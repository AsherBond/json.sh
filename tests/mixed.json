{
	"foo": ["bar", "baz"],
	"asdf": {
		"qwerty": ["hooah"]
	}
}
