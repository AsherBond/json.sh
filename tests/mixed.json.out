/foo/0 string bar
/foo/1 string baz
/asdf/qwerty/0 string hooah
